src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/ExceptionRecordProvider.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Maps;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition.YesNoFieldValues;
+
+import java.util.Map;
+
+import static java.util.Collections.emptyList;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition.ExceptionRecordFields.CASE_REFERENCE;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition.ExceptionRecordFields.DISPLAY_WARNINGS;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.definition.ExceptionRecordFields.OCR_DATA_VALIDATION_WARNINGS;
+
+@Service
+public class ExceptionRecordProvider {
```

Comment:
Sound very generic, while setting 'displayWarnings' to 'no' and clearing 'ocrDataValidationWarnings' is very specific and used in one particular scenario.

Reply:
same problem with `CcdCaseSubmitter`

Reply:
Renamed